docker/cortx-deploy/Dockerfile

```diff
@@ -50,8 +50,9 @@ COPY ./${ENVIRONMENT}/$OS.repo /etc/yum.repos.d/
 RUN if [[ "$ENVIRONMENT" = "opensource-ci" ]] && [[ $OS_TYPE == "rockylinux/rockylinux" ]]; then dnf install dnf-plugin-config-manager -y ;dnf config-manager --set-enabled powertools ; rm -rf /etc/yum.repos.d/$OS.repo ;fi
 
 # Install EPEL repo for opensource-ci
-RUN if [ "$ENVIRONMENT" = "opensource-ci" ] ; then yum install epel-release -y ; fi && yum clean all \
-&& rm -rf /var/cache/yum
+RUN yum remove -y libmnl \ 
```

Comment:
- This is similar to third-party package removal so let's follow the process defined at - https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/377226174/How+to+request+Third+party+package+addition+removal 
- Create a file named package-removal.txt at https://github.com/Seagate/cortx-re/tree/main/docker/cortx-deploy  and use it to list down packages to be removed from baseos. 
- `libmnl` also removes following packages to please list that also in above file. 

>     dracut-network-049-135.git20210121.el8.x86_64                                 
>     ethtool-2:5.8-5.el8.x86_64                                                    
>     iproute-5.9.0-4.el8.x86_64                                                    
>     kexec-tools-2.0.20-46.el8.x86_64